classes/services/PKPSchemaService.php

```diff
@@ -376,6 +376,11 @@ public function coerce($value, $type, $schema)
                 return $newArray;
             case 'object':
                 $newObject = []; // we handle JSON objects as assoc arrays in PHP
+                
+                if (isValidJson($value)) {
```

Comment:
Hmm, in which situation we need this?

Reply:
In our case. we are getting the data as JSON but here we try to access each inner prop of JSON data as array, it handle that case we for announcement files data are not converted to JSON to array .